Detectors/TRD/workflow/io/src/TRDCalibWriterSpec.cxx

```diff
@@ -14,6 +14,7 @@
 #include "Framework/InputSpec.h"
```

Comment:
@martenole Doe this warrant its own writer/reader can we not merge the for the vdrift calib into one, i.e., having one file like trdcalib.root?

Reply:
I am not sure if we need reader/writer at all?

Reply:
Hi @martenole, I am wondering: the gain calibration should run on the tracklet charges. If we restrict it to the digits, don't we then reduce the statistics a lot. When running on tracklets in the synchronous reco, we could obtain more statistics; however we need to merge the split charges before.

Reply:
I think in this implementation the digits are not used at all and the tracklet charges are used instead. This is how it was planned, right?

Reply:
Maybe I misunderstood your earlier comment. Indeed the tracklet charges should be used; thus to get more statistics, it might be better to not restrict ourselves to the digit events and therefore we do not need/use the writer and reader for the drift velocity; is that technically possible?

Reply:
Hi Ole, maybe we could just do something similar as for the t0 calibration; we wait until enough statistics is accumulated and the fits are very successful and then proceed with the gain calibration. Would that work?